devnotes/tpm.md

```diff
@@ -0,0 +1,228 @@
+This document is an attempt to gather information about the issues faced with
+TPMs on POWER9 and what are the potential solutions to them.
+
+## Some definitions
+
+* fTPM - Firmware-based TPM implementation
+
+## Problems
+
+### POWER9 cannot generate LPC TPM cycles
+
+This is a hardware limitation and cannot be worked around in firmware.
+
+The consequence is that regular LPC TPM cannot be used on POWER9 by directly
+connecting it to the TPM LPC header (one might wonder why have the LPC TPM
+header in the first place...).
+
+## Quick overview
+
+| Aspect\Approach | [I2C]        | [On BMC] | [FPGA LPC] | [FPGA SPI] | [On MCU]      | [LibreBMC]    | [SBE]    |
```

Comment:
I would be more specific here and make general division between dTPM and fTPM, all the other differentiators are mentioned  in the table


I'm not sure what `On MCU` means it